lib/stripe/list.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Stripe.List do
```

Comment:
Also, stripe considers it an object (it has an `"object" : "list"`) property, unlike, for example, `legal_entity` or `dob` which do not have such properties.